code/modules/spells/spell.dm

```diff
@@ -124,7 +124,7 @@ GLOBAL_LIST_INIT(spells, typesof(/obj/effect/proc_holder/spell)) //needed for th
 	var/nonabstract_req = FALSE //spell can only be cast by mobs that are physical entities
 	var/stat_allowed = FALSE //see if it requires being conscious/alive, need to set to 1 for ghostpells
 	var/phase_allowed = FALSE // If true, the spell can be cast while phased, eg. blood crawling, ethereal jaunting
-	var/antimagic_allowed = FALSE // If false, the spell cannot be cast while under the effect of antimagic
+	var/antimagic_flags = MAGIC_RESISTANCE // This determines what antimagic component can block the magic
```

Comment:
Swap this to an auto-doc please
(Also the wording is a bit strange - "what antimagic component can block"?)

Reply:
Sure thing https://github.com/tgstation/tgstation/pull/64124/commits/1ae70ad000d4c3040a14cf041de9c435446546b3